clang/lib/DPCT/ASTTraversal.cpp

```diff
@@ -8380,149 +8396,340 @@ void EventAPICallRule::findEventAPI(const Stmt *Node, const CallExpr *&Call,
 void EventAPICallRule::handleEventRecord(const CallExpr *CE,
```

Comment:
the function is quite big, how about we have two versoin of handleEventRecord,  one if for: event_profiling is enalbed,  other is the old one.

Reply:
Okay